### PR TITLE
add command runner workflow

### PR DESCRIPTION
**Pull request by ntduan**
fix issue #2

### CODE REVIEW

File: .github/workflows/command-runner.yml [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+        if: ${{fromJson(steps.prepare_command.outputs.result).command == 'run' || fromJson(steps.prepare_command.outputs.result).command == 'update'}}
+        run: ${{steps.run_command.outputs.result}} > ${{runner.temp}}/out.txt
+      - name: Check for file changes
+        id: check_changes
+        if: ${{fromJson(steps.prepare_command.outputs.result).command == 'update'}}
+        run: |
+          git fetch origin
+          if git diff --exit-code; then
+            echo "No changes detected" > result.txt
+            echo "::set-output name=changes_detected::false"
```
Comment (xlc) [inline .github/workflows/command-runner.yml:201]: I think this is the old deprecated syntax, should use the new one

**Review by xlc — CHANGES_REQUESTED** [review]
can we have most of the JS code in .js file and only load from .js file in yml file? it will be easier to read & test

**Comment by ntduan** [comment]
> can we have most of the JS code in .js file and only load from .js file in yml file? it will be easier to read & test

fixed

File: .github/workflows/command-runner.yml [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const payload = context.payload
+            const content = payload.comment ? payload.comment.body : ''
+            console.log(`content: ${content}`)
+            const lines = content.split('\n').map(line => line.trim()).filter(line => line.length > 0);
+            const [params, ...env] = lines;
+            const [, command, ...args] = params.split(/\s+/)
+            console.log(context)
```
Comment (xlc) [inline .github/workflows/command-runner.yml:27]:
```suggestion
```

**Pull request merged**